pkg/api/apiloader.go

```diff
@@ -4,6 +4,7 @@ import (
 	"encoding/json"
 	"io/ioutil"
 
+	"github.com/Azure/acs-engine/pkg/agentPoolOnlyApi/v20170831"
```

Comment:
thanks, makes sense.  I like the idea of having it under "api", but not having it as peer of cs type.